dockerless/start_backend.sh

```diff
@@ -2,4 +2,6 @@
 set -ex
 cd "$(dirname "$0")/.."
 
+export UNMUTE_CORS_ALLOW_ORIGINS="https://xzso00wrvev0el-3000.proxy.runpod.net/"
```

Comment:
I left this in as an example and reminder that it would need to be documented somewhere.